frontend/App.tsx

```diff
@@ -15,12 +15,17 @@ export function App(props) {
     <QueryClientProvider client={queryClient}>
       <head>
         <link rel="stylesheet" href="http://localhost:7700/main.css"></link>
+        <meta
```

Comment:
The changes to this file look like they are from your work on Google auth. Please remove them from this branch and only include them in the google auth branch